pyramid-app/{{ cookiecutter.slug }}/tests/pytest_plugins/pyramid.py

```diff
@@ -0,0 +1,14 @@
+{% if cookiecutter.get("postgres") == "yes" %}
+from os import environ
+
+{% endif %}
+import pytest
+
+
+@pytest.fixture
+def pyramid_settings():  # pragma: no cover
+{% if cookiecutter.get("postgres") == "yes" %}
+    return {"database_url": environ["DATABASE_URL"]}
+{% else %}
+    return {}
+{% endif %}
```

Comment:
Some basic Pyramid fixtures are also included out of the box. Again projects can override these in their `conftest.py`'s to customise them, or can add the entire file to their ignore list